metamorph/metamorph_plugin.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import logging.config
+import json
+import time
+import os
+
+import requests
+
+
+class MetamorphPlugin(object):
+
+    MINUTE = 60
```

Comment:
Guys, I partly agree with you. My coding experience telling me to avoid 'random' numbers in code. Also I believe it is more readable. 
`time.sleep(self.MINUTE)` implicitly is telling me that it will wait one minute. On the other hand `time.sleep(60)` is not telling me so.  

I know that the difference is not big but from my perspective it brings better readability and therefore it is worth doing it.